package.js

```diff
@@ -7,13 +7,13 @@ Package.describe({
 
 Package.onUse(function (api) {
   api.use([
-    //core
-    'templating@1.1.1',
+    // core
+    'ecmascript',
+    'templating',
 
     // atmosphere
-    'dispatch:spinner@0.0.4',
-    'flemay:less-autoprefixer@1.0.2',
-    'raix:handlebar-helpers@0.2.4'
+    'dispatch:spinner',
```

Comment:
You need to specify versions or add `api.versionsFrom('1.2')` at the top of `Package.onUse`